index.bs

```diff
@@ -5188,14 +5249,43 @@ when invoked, must run these steps:
 The <dfn method for=IDBTransaction>abort()</dfn> method, when invoked,
 must run these steps:
 
-1. If this [=/transaction=] is [=transaction/finished=], [=throw=]
-    an "{{InvalidStateError}}" {{DOMException}}.
+1. If this [=/transaction=]'s [=transaction/state=] is [=transaction/committing=]
+    or [=transaction/finished=],
+    then [=throw=] an "{{InvalidStateError}}" {{DOMException}}.
 
-1. Unset the [=/transaction=]'s [=transaction/active flag=] and run the
+1. Set the [=/transaction=]'s [=transaction/state=] to [=transaction/active=] and run the
```

Comment:
Shouldn't this say to set the transaction state to *inactive*?

Reply:
Yes, nice catch.